lib/action_account.rb

```diff
@@ -0,0 +1,148 @@
+require_relative './script_base'
+
+class ActionAccount < ScriptBase
```

Comment:
so I'm personally a big fan of delegation over inheritance.

I would consider something like

```
def script_helper
  @script_helper ||= ScriptHelper.new
end

def option_parser
  script_helper.option_parser
end
```

etc

and then we'd duplicate the constructors (those are relatively simple)